### DISCUSSION TITLE
联票抢票失败

### DISCUSSION THREAD

**Discussion by zgbsr** [mikumifa/biliTickerBuy#323] [提问专区]
参照了前面老哥的流程，准时抢，1秒进验证码，3秒过，之后看着都挺顺利的，但8秒报了一条慢一点，10秒token过期，11秒报429后就售罄了

**Comment by zgbsr** [comment]
求各位大佬指导，个人盲猜是票数太少加上300ms还不够快。。

**Reply by lurencc123** [reply]
可以继续抢吗？一直等，有的人不付款会返回票池吗？

**Comment by JYK89** [comment]
秒没 我还开了自动验证和手动验证 都抢不到
